metadata/2024-07-23-enable-fa-migration/enable_coin_to_fa_migration.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable the mapping from coin to fungible asset, including APT",
+  "description": "Enables the changes in AIP-63 https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-63.md",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
```

Comment:
source code url should be the .move file in this repo, the script for this proposal

Reply:
let me change